src/components/page/writedReviewPage.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { useQuery } from 'react-query';
+import { getWrtiedReview } from '../../apis/writedReview';
+import WritedReviewTemplate from '../template/writedReviewTemplate';
+
+function WritedReviewPage() {
+  const [page, setPage] = useState(1);
+  const limits = 8;
+  const { data, isLoading, isFetching } = useQuery({
+    queryKey: [`getWrtiedReview?cursor=${1 + 8 * (page - 1)}&limits=${limits}`],
+    queryFn: () => getWrtiedReview(1 + 8 * (page - 1), limits),
+  });
+  const onHandleChangePage = (type: 'right' | 'left') => {
+    if (type === 'right') {
+      setPage(() => page + 1);
```

Comment:
```suggestion
      setPage((prev) => prev + 1);
```
prev를 사용하시면 더 안전합니다.